diff --git a/node_modules/@imagemagick/magick-wasm/dist/index.mjs b/node_modules/@imagemagick/magick-wasm/dist/index.mjs
index 2a43ef4..2ef32fb 100644
--- a/node_modules/@imagemagick/magick-wasm/dist/index.mjs
+++ b/node_modules/@imagemagick/magick-wasm/dist/index.mjs
@@ -95,9 +95,19 @@ class Hi {
    */
   message;
 }
+
 var Fi = (() => {
   var t = typeof document < "u" && document.currentScript ? document.currentScript.src : void 0;
   return function(a = {}) {
+
+    // hack in worker
+    if(typeof window === "undefined"){
+      var window = {
+        crypto,
+        prompt: console.log
+      }
+    }
+
     var e = a, l, m;
     e.ready = new Promise((n, r) => {
       l = n, m = r;
